packages/sdk/src/swap/types.ts

```diff
@@ -51,7 +51,7 @@ export interface QuoteResponse
 
 export interface DepositAddressRequest extends QuoteRequest {
   destAddress: string;
-  ccmMetadata?: CcmMetadata;
+  ccmParams?: CcmParams;
```

Comment:
this is a breaking change - we need to fall back to `ccmMetadata` where we consume this type